homeassistant/brands/nextcloud.json

```diff
@@ -0,0 +1,5 @@
+{
+  "domain": "nextcloud",
+  "name": "Nextcloud",
+  "integrations": ["caldav", "nextcloud"]
```

Comment:
You're right but what's the correct way to give the users a hint that they could also use caldav with their Nextcloud server?

Reply:
There are multiple solutions, for example, the nextcloud integration could provide that calendar functionality (so, it just works without the need for other integrations).

Another solution would be providing a virtual integration binding to caldav, but IMHO that isn't really fitting in this case.

../Frenkc

Reply:
The first solution would be nice, but would also mean to implement caldav a second time. Or is there a way to use the other integration directly.

The second approach could also be a simple solution like this. But this would mean you have to set the url and credentials a second time what is technically not necessary.
```json
{
  "domain": "nextcloud_calendars",
  "name": "Nextcloud Calendars",
  "integration_type": "virtual",
  "supported_by": "caldav",
}
```